style.css

```diff
@@ -440,14 +522,19 @@ fieldset {
   }
 
   .ellipse_18 {
-    background-image: url('circle_1_header.png');
+    background-image: url('/images/circle_1_header.png');
     background-repeat: no-repeat;
     position: absolute;
     top: 1%;
     width: 284px;
     height: 284px;
   }
 
+  /* .imagePlaceholder {
+    width: 150%;
+    height: 100%;
+  } */
+
   .menu_items {
     display: flex;
```

Comment:
- [ ] Kindly remove this property for display: flex ,so in desktop version the menu-items not displayed

Reply:
thanks but I thought this is strictly for mobile view. I will handle it when it gets to the desktop version.